plugin.audio.rco/rco.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# Step 1 - load kodi core support and setup the environment
+import os
+import xbmcplugin, xbmcaddon
+import xbmc, xbmcgui
+import sys
+import urllib2
+import re
+import HTMLParser
+
+xbmc.log("plugin.video.rco:: Starting Addon")
+
+
+# magic; id of this plugin - cast to integer
+thisPlugin = int(sys.argv[1])
+
+settings = xbmcaddon.Addon(id='plugin.audio.rco')
```

Comment:
please remove the id kwargs. You can simply use `xbmcaddon.Addon()`.The id is only necessary when you try to access other addons from your addon.